src/pages/graphql/schema/attributes/queries/attributes-form.md

```diff
@@ -12,6 +12,8 @@ The `attributesForm` query retrieves EAV attributes associated with customer and
 
 These forms are visible when using the Admin to create or edit a customer or customer address address (**Stores** > Attributes > **Customer** or **Customer Address**).
 
+**Note:** For `region_id` and `country_id` attributes information use queries available in the `DirectoryGraphQl` module.
```

Comment:
Replace "**Note:**" with the following lines (including one blank line)

```
<InlineAlert variant="info" slots="text" />

```

Change the text of the note to something like

Use the [`country`](../../store/queries/country.md) and [`countries`](../../stores/queries/countries.md) mutations to retrieve information about the `region_id` and `country_id` attributes.